.github/workflows/basic.yml

```diff
@@ -86,7 +86,37 @@ jobs:
         run: ninja gen_docs
         working-directory: build
 
-        
+  cppcheck:
+    name: Check C++ linking with example program
+    runs-on: ubuntu-latest
+    container: openquantumsafe/ci-ubuntu-latest:latest
+    env:
+      KEM_NAME: ml_kem_768
```

Comment:
Do we need this environment variable? I think we only need the SIG_NAME for this test.

Reply:
No, don't believe we do - I had left it in to be cautious, but it shouldn't be necessary.